news/2022/2022-12-01-osumonthly-5.md

```diff
@@ -107,8 +107,6 @@ Word on the street is that there will be more Hush-Hush medals coming soon, so i
 
 ### SillyFanGirl's BR eSports nomination
 
-![https://twitter.com/premioesportsbr/status/1593590309631295488?s=20&t=1thfKY6-BpTCFDsbtltDdw](https://pbs.twimg.com/profile_banners/983759809156669440/1663599648/1500x500)
```

Comment:
this is why you don't hotlink images aaaaaaaaa

archive here https://web.archive.org/web/20230108000953im_/https://i.ppy.sh/2e2651e1d1b90751eb22f84336e2eb077016019d/68747470733a2f2f7062732e7477696d672e636f6d2f70726f66696c655f62616e6e6572732f3938333735393830393135363636393434302f313636333539393634382f3135303078353030

Reply:
Yea dunno how this got through i was thinking of removing it entirelt but luckily you found an archive. I will add this tomorrow